src/lib.rs

```diff
@@ -267,10 +268,18 @@ pub trait Probe {
 /// * Can produce and merge transparent snapshots
 #[repr(C)]
 pub struct ModalityProbe<'a> {
+    fingerprint: u32,
+    /// Pad fingerprint up to 8 bytes for consistent history offset
```

Comment:
Here "consistent" means "with the same field offset on both 32-bit and 64-bit architectures", right?

Reply:
If so, recommend adding some detail to that effect in the doc-comment.